utils/check-style/aspell-ignore/en/aspell-dict.txt

```diff
@@ -1,4 +1,4 @@
-personal_ws-1.1 en 543
+personal_ws-1.1 en 2586 
```

Comment:
Do you think this change is OK?

Reply:
So this seems to change the encoding. What did 543 stand for, what does 2586 stand for? To be on the safe side, please leave this change out or push it in a separate PR.

Reply:
Removed, it works without it. Thanks!